src/nets/e2e_asr_th.py

```diff
@@ -14,6 +14,7 @@
 
 import chainer
 import numpy as np
+import random
```

Comment:
seed?

Reply:
Thanks for pointing it out Shinji. I did not focus on the seed before and now after ur message I ran some experiments and found the results vary with different seeding. Till now I was using default system settings for seeding.
Tested with voxforge - Italian

| seed | % CER |
|--------|-----------|
| baseline| 14.8|
| 1 | 15.0 |
| 2 | 15.0|
|3 | 15.1|
|4 |14.5 |
|5| 15.6 |

Reply:
I think this change happens unfortunately, and we may not have a way to avoid it. We can leave it as it is now.

> So, will it be better if we set seeding as a parameter ?

Some recipe has an option to accept the seed, e.g., 
https://github.com/espnet/espnet/blob/d3cf8f0c2965d8cd3e9fc06b3d84b0af4318c9b8/egs/csj/asr1/run.sh#L18
We could make this as a default, but we may not have to do it for all recipes for now.